matlab/spherical-traversal/angular_hit.m

```diff
@@ -76,71 +97,95 @@
 else
     intersect_max = 0;
 end
-
+% Compute inverses of slopes
 if ~approximatelyEqual(ray_direction(1),0.0,1e-12,1e-8)
     inv_dir = 1/ray_direction(1);
 else
     inv_dir = 1/ray_direction(2);
 end
-
-if intersect_max == 1
+% Compute intersection times
+if intersect_max == 1 || collinear_max == 1
```

Comment:
so ```t_max``` is initialized if either of these are 1. What is its value if neither of these are 1?

Reply:
They should probably be initialized to t_end.

Reply:
Ok I leave mine defaulted to 0.0, and only change in this case. Either 0.0 or t_end should work though.